client/src/partials/admin/creditor_groups/creditor_groups.list.html

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-8 col-md-offset-2">
+
+      <div ng-repeat="cg in CreditorGroupCtrl.creditorGroupList | orderBy : cg.name">
+
+        <div class="panel panel-default" style="margin-bottom: 5px;" data-group-entry>
+          <div class="panel-body">
+
+            <h4>
+              <span ng-if="cg.locked" class="fa fa-lock text-danger"></span> <b>{{ cg.name }}</b>
+              <a style="text-decoration: none;"
```

Comment:
In my opinion, it would look nicer if the `update` and `delete` links were pulled right.  Like this:
```html
<span class="pull-right">
  <a ...></a>
  <a ...></a>
</span>
```
What do you think?